tests/README.md

```diff
@@ -0,0 +1,53 @@
+# Testing the tar Application
+
+This directory contains tests for the `tar` command-line utility.
+
+## Philosophy
+
+The `tar` utility is built on top of the `tar-rs` library. Because of this, we split our testing into two distinct areas:
+
+1.  **The Library (`tar-rs`)**: This is where we test the nitty-gritty details of the tar format. If you want to verify that permissions are preserved correctly, that long paths are handled according to the UStar spec, or that unicode filenames are encoded properly, those tests belong in `tar-rs/tests/`.
```

Comment:
I don't know if it's a planned refactoring, but currently there is no `tar-rs/tests` folder.